turbo.json

```diff
@@ -37,12 +37,12 @@
     },
     "build:scaffold-react": {
       "outputs": ["dist/**"],
-      "dependsOn": ["^build:scaffold-utils"],
+      "dependsOn": ["^build:scaffold-utils", "^build:scaffold", "^build:siwe"],
       "cache": false
     },
     "build:scaffold-vue": {
       "outputs": ["dist/**"],
-      "dependsOn": ["^build:scaffold-utils"],
+      "dependsOn": ["^build:scaffold-utils", "^build:scaffold", "^build:siwe"],
```

Comment:
I think `"^build:scaffold"` would not be needed given it already depends on `"^build:scaffold-utils"`

Reply:
if I remove them it breaks the build, seems like I need to explicitly declared all the packages is dependent on